containers/operator/Dockerfile

```diff
@@ -28,9 +28,11 @@ RUN dep ensure -vendor-only
 
 COPY legion/operator/pkg/    pkg/
 COPY legion/operator/cmd/    cmd/
+COPY legion/operator/docs/    docs/
```

Comment:
What is "docs"?

Reply:
It is the autogenerated swagger integration. Build will be failed without this dir